spec/models/casa_case_spec.rb

```diff
@@ -75,7 +76,7 @@
 
     context "when volunteer has no case assignments" do
       it "returns all cases in volunteer's organization" do
-        expect(described_class.available_for_volunteer(volunteer)).to eq [casa_case2, casa_case3, casa_case1]
+        expect(described_class.available_for_volunteer(volunteer).ids).to eq [casa_case2.id, casa_case3.id, casa_case1.id]
```

Comment:
![specerror](https://user-images.githubusercontent.com/21108297/96821839-bb49f180-13f6-11eb-80c7-ee80d19749e5.png)
Some reason I'm getting this error if I don't change this - so I amended to fix.

Reply:
If this screenshot included the section under "Diff" maybe I could tell what was up (or maybe not) but this seems fine for now :)

Reply:
Argh, it must've been to do with some of the other tests I hadn't fixed yet. I can't replicate it. Sorry! Would you like me to revert that part in a separate PR or shall we leave it?

Reply:
Revert in separate PR would be great :)